content/en/docs/concepts/configuration/secret.md

```diff
@@ -28,8 +32,14 @@ a password, a token, or a key. Such information might otherwise be put in a
 Pod specification or in an image. Users can create secrets and the system
 also creates some secrets.
 
-To use a secret, a Pod needs to reference the secret.
-A secret can be used with a Pod in three ways:
+The `data` field in a secret contains key-value pairs, similar to a
+{{< glossary_tooltip text="ConfigMap" term_id="configmap" >}}. However, Secret
+objects are intended for confidential configuration information such as passwords
+or encryption keys.
+
+To use that confidential data, a Pod needs to reference the Secret in a container
+definition.
```

Comment:
This is inaccurate. Not all Secret are referenced in container definition. Right?

Reply:
Good point, this could be misleading.

Reply:
nit: You could write, 
However, Secrets are ... 
instead of
However, Secret objects

Reply:
@sftim, Have you resolved the question about lines 40-41?

Reply:
Yep, I think this is addressed.

Reply:
OK